docs/custom_installation.md

```diff
@@ -1,57 +1,61 @@
+      ```
+      ---
+      "url" : "https://splunk-dev.testing.local:8088/services/collector"
+      "token" : "13311780-EC29-4DD0-A796-9F0CDC56F2AD"
+      ```
+      (**Note**: If [Disaster Recovery](https://puppet.com/docs/pe/latest/dr_overview.html) is enabled you will need to ensure these settings exist on the Replica node as well. This is often done through the `PE HA Replica` node group.)
 
-8. Now it is time to roll these settings out to the fleet of to the Puppet Masters in the installation. For Puppet Enterprise users:
-	- Navigate to Classification -> PE Infrastructure -> PE Master
-	- Select Configuration
-	- Press Refresh to ensure the splunk_hec class is loaded
-	- Add new class `splunk_hec`
-	- From the `Parameter name` select at least `url` and `token` and provide the same attributes from the testing configuration file
-	- Optionally set `enable_reports` to `true` if there isn't another component managing the servers reports setting, otherwise manually add `splunk_hec` to the settings as described in the manual steps. Note that if `enable_reports` is set to `true`, then you can have the module automatically add the `splunk_hec` report processor to the servers reports setting by setting the `reports` parameter to the empty string, `''`.
-	- Commit changes and run Puppet. It is best to navigate to the PE Certificate Authority Classification gorup and run Puppet there first, before running Puppet on the remaining machines
+6. Run `puppet apply -e 'notify { "hello world": }' --reports=splunk_hec` from the Puppet Server, this will load the report processor and test your configuration settings without actually modifying your Puppet Server's running configuration. If you are using the Puppet Report Viewer app in Splunk then you will see the page update with new data. If not, perform a search by the `sourcetype` you provided with your HEC configuration.
 
-9. For Inventory support in the Puppet Report Viewer, see the [Fact Terminus Support](fact_terminus_support.md)
+7. If configured properly the Puppet Report Viewer app in Splunk will show 1 node in the `Overview` tab.
 
-### Manual steps:
+8. Now it is time to roll these settings out to the fleet of Puppet Servers in the installation. For PE users:
+  * In the [PE console](https://puppet.com/docs/pe/latest/console_accessing.html), navigate to `Node groups` and expand `PE Infrastructure`.
+  * Select `PE Master` and navigate to the `Classes` tab.
+  * Click **Refresh** to ensure that the `splunk_hec` class is loaded.
+  * Add new class `splunk_hec`.
+  * From the `Parameter` drop down list you will need to configure at least `url` and `token`, providing the same values from the testing configuration file.
+      * Optionally set `enable_reports` to `true` if there isn't another component managing the servers reports setting. Otherwise manually add `splunk_hec` to the settings as described in the [manual steps](#manual-steps) below.
```

Comment:
Quick question - did we want to remove the 'note' part in the prev readme about if enable_reports to true, it can add the splunk_hec report processor automatically? Does that not work anymore?